### PR TITLE
wine-tkg: Debian packaging on Arch

### PR DESCRIPTION
**Pull request by FLGX06**
This allows wine-tkg to create debian packages on arch
Can be enabled with _GENERATE_DEBIAN_PACKAGE in config

TODO: Support different prefixes than /usr
TODO FIX: Make wine64-tkg the default as "wine64", the problem is it causes wine to complain about wine64 (invalid ELF header I think)

I didn't fully test this, but winecfg and wine explorer.exe works.
Hope this will be useful for people using Debian-based distros

### CODE REVIEW

File: wine-tkg-git/wine-tkg-scripts/package-debian.sh [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+Architecture: all
+Maintainer: Tk-Glitch <ti3nou@gmail.com>
+Homepage: https://github.com/Frogging-Family/wine-tkg-git
+Provides: ${pkgname}, wine, wine-development, wine-staging, libwine, fonts-wine, wine32, wine64
+Conflicts: wine, libwine, fonts-wine, wine32, wine64
+Replaces: wine, wine-development, wine-staging, libwine, fonts-wine, wine32, wine64
+License: LGPL
+Depends: libc6 (>= 2.17), libfontconfig1 (>= 2.11), libfreetype6 (>= 2.2.1), libncurses5 (>= 6), libasound2 (>= 1.0.16), libgcc1 (>= 1:3.0), libglu1-mesa | libglu1, liblcms2-2 (>= 2.2+git20110628), libldap-2.4-2 (>= 2.4.7), libmpg123-0 (>= 1.6.2), libopenal1 (>= 1.14), libpcap0.8 (>= 0.9.8), libpulse0 (>= 0.99.1), libx11-6, libxext6, libxml2 (>= 2.9.0), ocl-icd-libopencl1 | libopencl1, ocl-icd-libopencl1 (>= 1.0) | libopencl-1.1-1, zlib1g (>= 1:1.1.4)
+Installed-Size: $(du -sb ${files} | awk '{printf "%1.0f\n",$1/1024}')
+Description: 🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸🐸
```
Comment (Tk-Glitch) [inline wine-tkg-git/wine-tkg-scripts/package-debian.sh:87]: The description found in the PKGBUILD could be used here as well with minor edits - to be a bit less sketchy. Maybe with a few frogs still :frog:

**Comment by Tk-Glitch** [comment]
Also about that wine64 situation, what is it complaining about? I would have expected the WINEDLLPATH export to also affect `wine64` getting triggered by running `wine`.

File: wine-tkg-git/wine-tkg-scripts/package-debian.sh (hunk 1 above)
Comment (FLGX06) [reply wine-tkg-git/wine-tkg-scripts/package-debian.sh]: Changed, I was happy that it works and forgot to change it :frog:

**Comment by FLGX06** [comment]
So yeah, this should be ready as discussed on Discord without the `wine64` wrapper being the default.

**Comment by Tk-Glitch** [comment]
Also enforcing it to disabled for proton-tkg will be required unless you have plans to add proton-tkg support later on. Disabling it in the meantime seems reasonable still.

**Comment by FLGX06** [comment]
Done

**Comment by Tk-Glitch** [comment]
Thanks! Let's go.

**Pull request merged**